api/specs/webserver/v0/components/schemas/project-v0.0.1-converted.yaml

```diff
@@ -51,18 +51,19 @@ properties:
           enum:
```

Comment:
the whole thing being under x-patternProperties, everything below that is not used. by the openapi validator..